html/changelog.html

```diff
@@ -56,6 +56,14 @@
 <!-- DO NOT REMOVE OR MOVE THIS COMMENT! THIS MUST BE THE LAST NON-EMPTY LINE BEFORE THE LOGJS #ADDTOCHANGELOGMARKER# -->
 
 <div class='commit sansserif'>
+	<h2 class='date'>15 September 2015</h2>
+	<h3 class='author'>Memendia updated:</h3>
+	<ul class='changes bgimages16'>
+		<li class='rscadd'>Ported Crystal's PixelProjectiles!</li>
```

Comment:
you're a fucking faggot nex, and you never stopped being one, now port this:
https://github.com/tgstation/-tg-station/pull/4477